### DISCUSSION TITLE
关于暗黑模式下字体颜色

### DISCUSSION THREAD

**Discussion by zkwolf** [ant-design/ant-design#29263] [General]
建议将暗黑模式下(即dark.less中)@text-color变量从fade(@white, 85%)变成fade(@white, 65%)

1.相关修改：#25923 #25630
2.目前dark.less中默认@text-color的值为fade(@white, 85%)，但是官网在处理暗黑模式的时候替换变量替换成了fade(@white, 65%)，如果单独用dark-theme.js中变量替换的话字体颜色会和官网不一致
3.如果采用目前的fade(@white, 85%)，暗黑模式下部分组件，比如datepicker中对比度有些过高，字体颜色有些刺眼
4.即使换回fade(@white, 65%)，和默认的component-background变量测试下，其对比度依然符合WCAG2.0标准
![image](https://user-images.githubusercontent.com/8555127/107113610-fd5e4c80-689a-11eb-8222-bf433be50159.png)

**Comment by yoyo837** [comment]
改为Issue⑧

**Reply by zkwolf** [reply]
ok
